lib/quic.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const {
+  createSocket
+} = require('internal/quic/core');
+
+createSocket.createSocket = createSocket;
```

Comment:
It is not clear to me why we are setting this property. Should this be removed and the export be something like
```js
module.exports = {
  createSocket
}
```
And usage would become:
```js
const quic = require('quic');
const socket = quic.createSocket({ type: 'udp4', port: 1234 });
```